perfmetrics/scripts/hns_rename_folders_metrics/renaming_benchmark_test.py

```diff
@@ -17,6 +17,30 @@
 
 class TestRenamingBenchmark(unittest.TestCase):
 
+  def test_calculate_num_files(self):
```

Comment:
Add test case for nested folders.

Reply:
This function is only relevant for nested case and  takes in the folder structure JSON.So, I have updated the existing test.